.gitignore

```diff
@@ -3,3 +3,7 @@ env/
 
 __pycache__/
 peewee.db
+
+.idea
+.idea/
+docker-compose.yml
```

Comment:
is this `docker-compose.yml` a docker generated file?

Reply:
No.
i will remove it from gitignore in next update